sample/src/app/app.component.ts

```diff
@@ -20,21 +20,22 @@ import { UpboatsComponent } from './upboats/upboats.component';
     MessagingComponent,
     RemoteConfigComponent,
     StorageComponent,
-    UpboatsComponent
+    UpboatsComponent,
   ],
   template: `
     Hello World!
     <app-auth />
-    @defer (hydrate on idle) { <app-database /> } @placeholder { Database! &hellip; }
-    @defer (hydrate on idle) { <app-firestore /> } @placeholder { Firestore! &hellip; }
-    @defer (hydrate on idle) { <app-functions /> } @placeholder { Functions! &hellip; }
-    @defer (hydrate on idle) { <app-messaging /> } @placeholder { Messaging! &hellip; }
-    @defer (hydrate on idle) { <app-remote-config /> } @placeholder { Remote Config! &hellip; }
-    @defer (hydrate never) { <app-storage /> } @placeholder { Storage! &hellip; }
-    @defer (hydrate on idle) { <app-upboats /> } @placeholder { &hellip; }
+    @defer (hydrate on idle) { <app-database /> } @placeholder { Database!
+    &hellip; } @defer (hydrate on idle) { <app-firestore /> } @placeholder {
+    Firestore! &hellip; } @defer (hydrate on idle) { <app-functions /> }
+    @placeholder { Functions! &hellip; } @defer (hydrate on idle) {
+    <app-messaging /> } @placeholder { Messaging! &hellip; } @defer (hydrate on
+    idle) { <app-remote-config /> } @placeholder { Remote Config! &hellip; }
+    @defer (hydrate never) { <app-storage /> } @placeholder { Storage! &hellip;
+    } @defer (hydrate on idle) { <app-upboats /> } @placeholder { &hellip; }
```

Comment:
I'd recommend we start using `prettier` for formatting (something to think about)